sql/python/README.md

```diff
@@ -1,13 +1,16 @@
 # Python Code Template
```

Comment:
Since we are removing sql_data and the test, should we move the readme.md under submitter folder? Or we will rename python folder to submitter and move python code on level up